RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -113,6 +115,7 @@
    chargedHadronTrackResolutionFilter *
    BadChargedCandidateFilter*
    BadPFMuonFilter *
+   BadPFMuonDzFilter *
```

Comment:
adding to the end of the list would actually be better, but I couldn't figure out a more straightforward command than the following

```python
from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
run2_miniAOD_UL.toModify(metFilters, func=lambda t:t.insert(len(t._seq._collection), BadPFMuonDzFilter))
```

with ```len(t._seq._collection)``` it works, but requires a bit of internal member info.
@makortel @Dr15Jones is there a simpler syntax to append a module to a sequence in this context (with a python lambda)?

Reply:
How about
```python
run2_miniAOD_UL.toReplaceWith(metFilters, cms.Sequence(metFilters+BadPFMuonDzFilter))
```
?

Reply:
thanks, this should work as well.

I constrained myself to using an object modification instead of a replacement (perhaps I'm overthinking the costs of making a new object)